integration/kubernetes/init.sh

```diff
@@ -155,7 +155,10 @@ echo "Kubelet options:"
 sudo cat /var/lib/kubelet/kubeadm-flags.env
 sudo systemctl daemon-reload && sudo systemctl restart kubelet
 
-kubectl get nodes
+for i in $(seq 4)
```

Comment:
Nit: the commit's subject for files in integration/kubernetes is usually "k8s" or "integration/kubernetes". The "ci" component is misleading.

IIUC it runs `kubectl get nodes` as a mean to check if the kubelet was restarted correctly (the `sudo systemctl restart kubelet` command).  I wonder if there is a better approach.

Anyway for the current refactored code I see the problems:
 * the loop never fail. You need to check if `i` is `4` then kill the script (because I could not get the pods).
 * it should "sleep" first then check. Otherwise `i==4` won't do any difference as it will falls off the loop afterwards

Reply:
@snir911 just noticed it was merged...it will need a follow up PR

Reply:
Right! kid mistake @wainersm :P 
As for the first one, if getting the nodes will fail i expect also getting the pods afterwards will fail it